src/MedicalWorkers/Doctor.java

```diff
@@ -1,19 +1,28 @@
-package HospitalClassHierarchy;
+package MedicalWorkers;
 
-import java.util.Arrays;
 import java.util.Scanner;
 
+import MedicalForms.AppointmentForm;
+import MedicalForms.WellnessForm;
+import Patients.CheckupPatient;
+import Tools.Sphygmomanometer;
+import Tools.Stethoscope;
+import Tools.Thermometer;
+import Tools.Tools;
+
 public class Doctor {
 	private String branchOfHospital;
 	private String name;
 	private int Experience;
 	private boolean inResidency; 
+	private int DoctorID;
 
-	public Doctor(String branchOfHospital, String name, int Experience, boolean inResidency) {
+	public Doctor(String branchOfHospital, String name, int Experience, boolean inResidency, int DoctorID) {
 		this.branchOfHospital = branchOfHospital;
 		this.name = name;
 		this.inResidency = inResidency; 
 		this.Experience = Experience; 
+		this.DoctorID = DoctorID;
```

Comment:
fields like if be better generated and controlled only by a program. Users can only get an id if needed.